analyzers/rspec/cs/S108_c#.html

```diff
@@ -1,7 +1,7 @@
 <p>Most of the time a block of code is empty when a piece of code is really missing. So such empty block must be either filled or removed.</p>
 <h2>Noncompliant Code Example</h2>
 <pre>
-for (int i = 0; i &lt; 42; i++){}  // Empty on purpose or missing piece of code ?
+<code data-lang="text">for (int i = 0; i &lt; 42; i++){}  // Empty on purpose or missing piece of code ?</code>
```

Comment:
I think this should be `<code data-lang="csharp">`

Reply:
We can't fix these, the root cause is in the RSPEC repo:
https://github.com/SonarSource/rspec/blame/master/rules/S108/noncompliant.adoc

Reply:
Because the example is shared between multiple languages and the highlighting doesn't work as expected.